addons/website/static/src/components/editor/editor.js

```diff
@@ -78,9 +78,8 @@ export class WebsiteEditorComponent extends Component {
     }
 
     async quit(reload = true) {
```

Comment:
maybe its better to remove the param as well, the commit message is explicit enough